### DISCUSSION TITLE
Make scrollbars draggable from the extreme border of the screen

### DISCUSSION THREAD

**Discussion by CutterXYZ** [zen-browser/desktop#3952] [Ideas]
Hi,

There's a few pixels-wide border at the edge of the screen that prevents interacting with scrollbars from the edge. It would be faster to just sway the mouse to the right to click the scrollbar, rather than what we do now :
1. Move the cursor quickly to the extreme right of the screen
2. Then move it left precisely by a few pixels <- this is unnecessary

Please make the scrollbars draggable from the borders.

See [Fitt's Law](https://en.wikipedia.org/wiki/Fitts%27s_law)

**Comment by new-creation7** [comment]
There is a Zen Mod named 'No Gapps'

https://zen-browser.app/mods/bfcc400a-4ecb-4752-bfd2-a68f116a2722
